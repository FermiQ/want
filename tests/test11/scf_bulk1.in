Pt 1 atom

   &CONTROL
      calculation = 'scf',
      restart_mode='from_scratch',
      prefix='pt1',
      nstep=50,
      tstress = .FALSE.,
      tprnfor = .TRUE.,
      pseudo_dir = '../pseudo/',
      outdir = './SCRATCH'
   /
   &SYSTEM
      ibrav=8, 
      celldm(1)=18.0,
      celldm(2)=1.0,
      celldm(3)=.2530134320,    ! 2.410 Ang = 4.5542417754 Bohr
      nat=1, 
      ntyp=1,
      ecutwfc=22.0,
      ecutrho=176.0,
      occupations='smearing',
      smearing='gaussian',
      degauss=0.01,
   /
   &ELECTRONS
      electron_maxstep = 50,
      mixing_mode = 'plain',
      mixing_beta = 0.3,
      conv_thr =  1.0d-8,
   /
ATOMIC_SPECIES
  Pt  195.08    Pt.pz-rrkjus.UPF
ATOMIC_POSITIONS {angstrom}
Pt    0.000    0.000   0.000
K_POINTS {automatic}
   1 1 16    0  0  1

