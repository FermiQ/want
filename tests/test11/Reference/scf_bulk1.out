
     Program PWSCF     v.4.1.2  starts ...
     Today is  5Apr2010 at 18:13:47 

     Parallel version (MPI)

     Number of processors in use:       6
     R & G space division:  proc/pool =    6

     For Norm-Conserving or Ultrasoft (Vanderbilt) Pseudopotentials or PAW

     Current dimensions of program pwscf are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Waiting for input...
     file Pt.pz-rrkjus.UPF: wavefunction(s)  5D renormalized

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a parallel distributed memory algorithm will be used,
     eigenstates matrixes will be distributed block like on
     ortho sub-group =    2*   2 procs


     Planes per process (thick) : nr3 = 20 npp =   4 ncplane = 6400
     Planes per process (smooth): nr3s= 15 npps=   3 ncplanes= 2916
 
     Proc/  planes cols     G    planes cols    G      columns  G
     Pool       (dense grid)       (smooth grid)      (wavefct grid)
        1     4    752     9702    3    376     3428    122      630
        2     4    752     9702    3    376     3428    124      630
        3     3    752     9702    3    375     3427    124      630
        4     3    752     9702    2    375     3427    124      630
        5     3    752     9702    2    375     3421    124      630
        6     3    753     9701    2    376     3422    123      629
     tot     20   4513    58211   15   2253    20553    741     3779
 


     bravais-lattice index     =            8
     lattice parameter (a_0)   =      18.0000  a.u.
     unit-cell volume          =    1475.5743 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        10.00
     number of Kohn-Sham states=            9
     kinetic-energy cutoff     =      22.0000  Ry
     charge density cutoff     =     176.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PZ   NOGX NOGC (1100)

     celldm(1)=  18.000000  celldm(2)=   1.000000  celldm(3)=   0.253013
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  1.000000  0.000000  0.000000 )  
               a(2) = (  0.000000  1.000000  0.000000 )  
               a(3) = (  0.000000  0.000000  0.253013 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  3.952359 )  


     PseudoPot. # 1 for Pt read from file Pt.pz-rrkjus.UPF
     Pseudo is Ultrasoft, Zval = 10.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 1547 points,  3 beta functions with: 
                l(1) =   1
                l(2) =   2
                l(3) =   2
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Pt            10.00   195.08000     Pt( 1.00)

     16 Sym.Ops. (with inversion)


   Cartesian axes

     site n.     atom                  positions (a_0 units)
         1           Pt  tau(  1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=    8  gaussian broad. (Ry)=  0.0100     ngauss =   0
                       cart. coord. in units 2pi/a_0
        k(    1) = (   0.0000000   0.0000000   0.1235112), wk =   0.2500000
        k(    2) = (   0.0000000   0.0000000   0.3705337), wk =   0.2500000
        k(    3) = (   0.0000000   0.0000000   0.6175561), wk =   0.2500000
        k(    4) = (   0.0000000   0.0000000   0.8645786), wk =   0.2500000
        k(    5) = (   0.0000000   0.0000000   1.1116011), wk =   0.2500000
        k(    6) = (   0.0000000   0.0000000   1.3586235), wk =   0.2500000
        k(    7) = (   0.0000000   0.0000000   1.6056460), wk =   0.2500000
        k(    8) = (   0.0000000   0.0000000   1.8526684), wk =   0.2500000

     G cutoff = 1444.4348  (  58211 G-vectors)     FFT grid: ( 80, 80, 20)
     G cutoff =  722.2174  (  20553 G-vectors)  smooth grid: ( 54, 54, 15)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.06 Mb     (    428,   9)
        NL pseudopotentials             0.08 Mb     (    428,  13)
        Each V/rho on FFT grid          0.39 Mb     (  25600)
        Each G-vector array             0.07 Mb     (   9702)
        G-vector shells                 0.02 Mb     (   2557)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         0.24 Mb     (    428,  36)
        Each subspace H/S matrix        0.02 Mb     (     36,  36)
        Each <psi_i|beta_j> matrix      0.00 Mb     (     13,   9)
        Arrays for rho mixing           3.12 Mb     (  25600,   8)

     Initial potential from superposition of free atoms

     starting charge    9.99989, renormalised to   10.00000

     negative rho (up, down):  0.846E-05 0.000E+00
     Starting wfc are    6 atomic +    3 random wfc

     total cpu time spent up to now is      0.69 secs

     per-process dynamical memory:    31.9 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  7.4

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  9.21E-04,  avg # of iterations =  1.0

     negative rho (up, down):  0.409E-05 0.000E+00

     total cpu time spent up to now is      1.48 secs

     total energy              =     -52.43105394 Ry
     Harris-Foulkes estimate   =     -52.46635349 Ry
     estimated scf accuracy    <       0.09338659 Ry

     iteration #  2     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  9.34E-04,  avg # of iterations =  1.1

     negative rho (up, down):  0.109E-04 0.000E+00

     total cpu time spent up to now is      1.79 secs

     total energy              =     -52.43844953 Ry
     Harris-Foulkes estimate   =     -52.44126596 Ry
     estimated scf accuracy    <       0.00931591 Ry

     iteration #  3     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  9.32E-05,  avg # of iterations =  2.6

     negative rho (up, down):  0.954E-04 0.000E+00

     total cpu time spent up to now is      2.12 secs

     total energy              =     -52.43839419 Ry
     Harris-Foulkes estimate   =     -52.43923497 Ry
     estimated scf accuracy    <       0.00223368 Ry

     iteration #  4     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.23E-05,  avg # of iterations =  2.6

     negative rho (up, down):  0.594E-04 0.000E+00

     total cpu time spent up to now is      2.46 secs

     total energy              =     -52.43848624 Ry
     Harris-Foulkes estimate   =     -52.43848999 Ry
     estimated scf accuracy    <       0.00002073 Ry

     iteration #  5     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.07E-07,  avg # of iterations =  2.9

     negative rho (up, down):  0.330E-04 0.000E+00

     total cpu time spent up to now is      2.90 secs

     total energy              =     -52.43848618 Ry
     Harris-Foulkes estimate   =     -52.43849092 Ry
     estimated scf accuracy    <       0.00000914 Ry

     iteration #  6     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  9.14E-08,  avg # of iterations =  2.0

     negative rho (up, down):  0.201E-04 0.000E+00

     total cpu time spent up to now is      3.24 secs

     total energy              =     -52.43848746 Ry
     Harris-Foulkes estimate   =     -52.43848786 Ry
     estimated scf accuracy    <       0.00000115 Ry

     iteration #  7     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.15E-08,  avg # of iterations =  2.0

     negative rho (up, down):  0.116E-04 0.000E+00

     total cpu time spent up to now is      3.57 secs

     total energy              =     -52.43848776 Ry
     Harris-Foulkes estimate   =     -52.43848771 Ry
     estimated scf accuracy    <       0.00000011 Ry

     iteration #  8     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.07E-09,  avg # of iterations =  2.0

     negative rho (up, down):  0.656E-05 0.000E+00

     total cpu time spent up to now is      3.90 secs

     total energy              =     -52.43848782 Ry
     Harris-Foulkes estimate   =     -52.43848778 Ry
     estimated scf accuracy    <       0.00000004 Ry

     iteration #  9     ecut=    22.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  3.91E-10,  avg # of iterations =  1.2

     negative rho (up, down):  0.241E-06 0.000E+00

     total cpu time spent up to now is      4.20 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.1235 (  2591 PWs)   bands (ev):

   -11.5637  -6.4245  -6.3070  -6.3069  -4.2927  -4.2927  -1.1902  -1.1902
     1.2342

          k = 0.0000 0.0000 0.3705 (  2567 PWs)   bands (ev):

   -11.0549  -6.2868  -6.2386  -6.2384  -4.8757  -4.8757  -0.7061  -0.7061
     1.4311

          k = 0.0000 0.0000 0.6176 (  2583 PWs)   bands (ev):

   -10.0673  -6.1100  -6.1098  -6.0167  -5.7080  -5.7080   0.0027   0.0027
     1.8247

          k = 0.0000 0.0000 0.8646 (  2595 PWs)   bands (ev):

    -8.6675  -6.5741  -6.5741  -5.9374  -5.9372  -5.6192   0.8315   0.8315
     2.4156

          k = 0.0000 0.0000 1.1116 (  2571 PWs)   bands (ev):

    -7.3617  -7.3617  -7.0067  -5.7443  -5.7442  -5.0569   1.7729   1.7729
     3.2048

          k = 0.0000 0.0000 1.3586 (  2567 PWs)   bands (ev):

    -8.0023  -8.0023  -5.5819  -5.5597  -5.5596  -3.9996   2.8329   2.8329
     4.1959

          k = 0.0000 0.0000 1.6056 (  2558 PWs)   bands (ev):

    -8.4519  -8.4519  -5.4142  -5.4141  -4.9103  -2.1735   3.9886   3.9886
     5.3458

          k = 0.0000 0.0000 1.8527 (  2534 PWs)   bands (ev):

    -8.6830  -8.6830  -5.3337  -5.3337  -4.6358  -0.4247   3.8629   5.0539
     5.0539

     the Fermi energy is    -4.9222 ev

!    total energy              =     -52.43848791 Ry
     Harris-Foulkes estimate   =     -52.43848783 Ry
     estimated scf accuracy    <          7.3E-10 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =     -89.31579506 Ry
     hartree contribution      =      47.83293116 Ry
     xc contribution           =      -8.66314708 Ry
     ewald contribution        =      -2.29024915 Ry
     smearing contrib. (-TS)   =      -0.00222777 Ry

     convergence has been achieved in   9 iterations

     Forces acting on atoms (Ry/au):

     atom   1 type  1   force =     0.00000000    0.00000000    0.00000000

     Total force =     0.000000     Total SCF correction =     0.000000

     Writing output data file pt1.save
 
     PWSCF        :     4.50s CPU time,    6.97s wall time

     init_run     :     0.62s CPU
     electrons    :     3.51s CPU
     forces       :     0.21s CPU

     Called by init_run:
     wfcinit      :     0.08s CPU
     potinit      :     0.12s CPU

     Called by electrons:
     c_bands      :     2.12s CPU (      10 calls,   0.212 s avg)
     sum_band     :     0.85s CPU (      10 calls,   0.085 s avg)
     v_of_rho     :     0.05s CPU (      10 calls,   0.005 s avg)
     newd         :     0.40s CPU (      10 calls,   0.040 s avg)
     mix_rho      :     0.04s CPU (      10 calls,   0.004 s avg)

     Called by c_bands:
     init_us_2    :     0.05s CPU (     176 calls,   0.000 s avg)
     cegterg      :     2.05s CPU (      80 calls,   0.026 s avg)

     Called by *egterg:
     h_psi        :     1.59s CPU (     287 calls,   0.006 s avg)
     s_psi        :     0.03s CPU (     287 calls,   0.000 s avg)
     g_psi        :     0.02s CPU (     199 calls,   0.000 s avg)
     cdiaghg      :     0.30s CPU (     271 calls,   0.001 s avg)

     Called by h_psi:
     add_vuspsi   :     0.03s CPU (     287 calls,   0.000 s avg)

     General routines
     calbec       :     0.07s CPU (     375 calls,   0.000 s avg)
     cft3s        :     1.80s CPU (    4452 calls,   0.000 s avg)
     interpolate  :     0.04s CPU (      20 calls,   0.002 s avg)
     davcio       :     0.00s CPU (     256 calls,   0.000 s avg)
 
     Parallel routines
     fft_scatter  :     0.56s CPU (    4452 calls,   0.000 s avg)
